python/ray/tests/test_worker_capping.py

```diff
@@ -140,7 +139,8 @@ def foo():
     block_driver_ref = block_driver.acquire.remote()
 
     # Both tasks should be running, so the driver should be unblocked.
-    ready, not_ready = ray.wait([block_driver_ref], timeout=1)
+    timeout_value = 5 if sys.platform == "win32" else 1
```

Comment:
can we just make this 5 on all platforms?

Reply:
Sure. I thought if we keep it 5 for Linux too then we might allow some slowdowns to go unnoticed for Linux in distant future.

Reply:
Should I default to 5 for all platforms? Please let me know.